components/managedidentity/locals.tf

```diff
@@ -1,5 +1,6 @@
 locals {
-  mi_environment = var.env == "ptlsbox" ? "cftsbox-intsvc" : var.env == "ptl" ? "cftptl-intsvc" : var.env == "sbox" ? "sandbox" : var.env == "stg" ? "aat" : var.env == "dev" ? "preview" : var.env == "test" ? "perftest" : var.env == "jenkins_prod" ? "prod" : var.env == "jenkins_sbox" ? "sandbox" : var.env
+  mi_environment = var.env == "ptlsbox" ? "cftsbox-intsvc" : var.env == "ptl" ? "cftptl-intsvc" : var.env == "sbox" ? "sandbox" : var.env == "stg" ? "aat" : var.env == "dev" ? "preview" : var.env == "test" ? "perftest" : var.env
+  new_mi_environment = var.env == "ptlsbox" ? "cftsbox-intsvc" : var.env == "ptl" ? "cftptl-intsvc" : var.env == "sbox" ? "sds-jenkins-pipeline-metrics-sbox" : var.env == "stg" ? "aat" : var.env == "dev" ? "preview" : var.env == "test" ? "perftest" : var.env == "prod" ? "sds-jenkins-pipeline-metrics" : var.env
```

Comment:
Please avoid naming things *new*, it's not ideal for the future - and does this need reference to every other env too? I thought just prod/sbox